aii-dhcp/src/main/pan/quattor/aii/dhcp/config.pan

```diff
@@ -2,24 +2,31 @@
 # ${developer-info}
 # ${author-info}
 # ${build-info}
-template quattor/aii/${project.artifactId}/config;
+template quattor/aii/dhcp/config;
 
-include 'quattor/aii/${project.artifactId}/schema';
+include 'quattor/aii/dhcp/schema';
 
-# TFTP server
-# This is optional. Only it is necessary if your TFTP server is running on
-# a different machine than the DHCP server
-#
-# "/system/aii/dhcp/options/tftpserver" = "tftp.mydomain.org"
-#
+bind "/system/aii/discovery/dhcp" = structure_dhcp_module_info;
+
+prefix "/system/aii/discovery/dhcp";
+
+@documentation{
+    Enable the plugin
+}
+"enabled" = true;
```

Comment:
seriously backwards incompatible
i propose to set `"enabled" ?= false;`

Reply:
I disagree :-) When `quattor/aii/ks/config` is included, it makes sure `/system/aii/osinstall/ks` is not empty, so the plugin will actually be used.

This template does exactly the same. If I write `include 'quattor/aii/dhcp/config'`, I _want_ that to cause the plugin to be enabled. But, since the plugin does not have any plugin-wide configuration options at the moment, the `enabled` flag is used to serve this purpose.

You could call `enabled` as `an_arbitrary_value_to_ensure_the_path_exists`, but that is much harder to type :-)

Reply:
with current code, including `quattor/aii/dhcp/config` sets the discovery plugin, and effectively disables any dhcp configuration. i'm not sure it's the intention, at the very very least is backwards incompatible